src/addresses.config.ts

```diff
@@ -1,130 +1,194 @@
     tokenAddress: "0x1aBaEA1f7C830bD89Acc67eC4af516284b1bC33c",
+    network: Network.ETH,
   },
   {
+    assetType: AssetType.ERC20,
     label: "ETH",
     token: "WETH",
-    addresses: [wallets.CUSTODIAN_SAFE],
+    addresses: [wallets.RESERVE_MULTISIG_ETH, wallets.CUSTODIAN_ETH],
     tokenAddress: tokensAddresses.WETH_ON_ETH,
+    network: Network.ETH,
+    isWrappedAsset: true,
   },
   {
+    assetType: AssetType.ERC20,
     label: "BTC",
     token: "WBTC",
-    addresses: [wallets.CUSTODIAN_SAFE],
+    addresses: [wallets.CUSTODIAN_ETH],
     tokenAddress: tokensAddresses.WBTC_ON_ETH,
+    network: Network.ETH,
+    isWrappedAsset: true,
+  },
+  {
+    assetType: AssetType.ERC20,
+    label: "Staked agEUR",
+    token: "stEUR",
+    decimals: 18,
+    addresses: [wallets.RESERVE_MULTISIG_CELO],
+    tokenAddress: tokensAddresses.STEUR_ON_CELO,
+    network: Network.CELO,
+  },
+  {
+    assetType: AssetType.ERC20,
+    label: "Savings DAI",
+    token: "sDAI",
+    decimals: 18,
+    addresses: [wallets.RESERVE_MULTISIG_ETH],
+    tokenAddress: tokensAddresses.SAVINGS_DAI,
+    network: Network.ETH,
+  },
+  {
+    assetType: AssetType.ERC20,
+    label: "Lido Staked ETH",
+    token: "stETH",
+    decimals: 18,
+    addresses: [wallets.RESERVE_MULTISIG_ETH],
+    tokenAddress: tokensAddresses.LIDO_STAKED_ETH,
+    network: Network.ETH,
   },
 ]
 // WHEN Adding new TOKENS also update the TokenColor enum in PieChart.tsx
 
 export default ADDRESSES
 
-export function generateLink(token: Tokens, address: string) {
-  switch (token) {
-    case "CELO":
-      return `https://explorer.celo.org/address/${address}/coin_balances`
-    case "BTC":
-      return `https://blockchain.info/address/${address}`
-    case "ETH":
-      return `https://etherscan.io/address/${address}`
-    case "USDC":
-    case "EUROC":
-    case "DAI":
-    case "WBTC":
-    case "WETH":
-      return `https://etherscan.io/address/${address}`
-    case "cUSD in Curve Pool":
-    case "USDC in Curve Pool":
-      return `https://explorer.celo.org/mainnet/address/${address}/tokens#address-tabs`
+export type ReserveCryptoForDisplay = Omit<ReserveCrypto, "addresses"> & {
+  addresses: { address: string; symbol: Tokens }[]
+}
+
+export function generateLink(token: ReserveCrypto, address: string) {
+  if (token.assetType == AssetType.ERC20 || token.assetType == AssetType.Native) {
+    switch (token.token) {
+      case "CELO":
+        return `https://explorer.celo.org/address/${address}/coin_balances`
+      case "BTC":
+        return `https://blockchain.info/address/${address}`
+      case "ETH":
+        return `https://etherscan.io/address/${address}`
+      case "USDC":
+      case "EUROC":
+      case "DAI":
+      case "WBTC":
+      case "WETH":
+        return `https://etherscan.io/address/${address}`
+      case "stEUR":
+        return `https://explorer.celo.org/mainnet/address/${address}`
```

Comment:
we should add a case for sDai and stETh otherwise the addresses in the "Reserve Addresses" section won't send the user to the explorer